logging/Limiting_Access_to_Logs.md

```diff
@@ -15,101 +26,172 @@ only for a specified namespace.
+combination of the namespace name and the SAS Viya tenant name (if used). For example, if you ran the script and specified only `--namespace mynamespace`, the script would remove the Kibana-tenant space named `mynamespace`. If you ran the script and specified 
+`--namespace mynamespace --tenant mytenant1`, the script would remove the Kibana-tenant space named `mynamespace_mytenant1`. 
 
 ## Managing User Accounts for Restricted Access
 
-Use the `/logging/bin/user.sh` script to create or delete Kibana accounts 
-that have access to only the log messages and Kibana tenant space 
-associated with a specified namespace or SAS Viya tenant. You can use 
-the script to create user accounts for a Kibana tenant space anytime 
-after you have run the `onboard.sh` to implement access limitations for 
-the Kibana tenant space.
+### Introduction
+
+You can create or delete Kibana accounts 
+that have access to only the log messages and Kibana-tenant space 
+associated with a specified namespace or SAS Viya tenant. 
 
-This is the syntax for the script to add a user:
+### Creating User Accounts
+After you run the `onboard.sh` to implement access limitations for 
+the Kibana-tenant space, you can run 
+the  `/logging/bin/user.sh` script to create user accounts for a Kibana-tenant space. 
 
-```
-/logging/bin/user.sh CREATE --namespace <namespace>  [--tenant <tenant>]  [--user][<username>] [--password <password>]
-```
-The `namespace` parameter is required. It specifies the Kubernetes namespace to which the user should have access
 
-The `tenant` parameter specifies the SAS Viya tenant to which the user should have access.
+Here is the syntax for the script to add a user:
 
-If you do not specify a name for the `user` parameter, a user is 
+<pre>```
+/logging/bin/user.sh CREATE --namespace <i>namespace</i> [--tenant <i>tenant</i>] 
+[--user][<i>user_name</i>] [--password <i>password</i>]
+</pre>
+
+- *namespace* is required. It specifies the Kubernetes namespace to which the 
+  user can access.
+- *tenant* specifies the SAS Viya tenant to which the user can access.
+- *user* specifies a user name. If you do not specify a user name, a user is 
 created with the default name of `<Kibana tenant space name>_admin`. 
-For example, if you ran the script and specified `--namespace mynamespace --tenant mytenant1 --user`, the user would 
-be `mynamespace_mytenant1_admin`.
+For example, if you ran the script and specified 
+`--namespace mynamespace --tenant mytenant1 --user`, the user would be `mynamespace_mytenant1_admin`.
+
+You can create new users and link them to these access controls by running the 
+user.sh script.  For example, to create a user called "chen" and link the user to these 
+access controls, submit the following command:
 
-This is the syntax for the script to remove a user:
+<pre>
+/logging/bin/user.sh CREATE --user <i>user_name</i> --password <i>password</i> 
+--namespace _all_ --tenant _all_
```

Comment:
These are a special case, when both namespace and tenant are "\_all\_" the user will use the same, very broad, access controls as the `logadm` user.  There are valid reasons why a company would choose to create a new user with this same level of access.  However, I think the more common use-case is to tie the new user to the more narrow access controls limiting them to a specific namespace or namepace/tenant combination.  Given that, perhaps change the namespace and tenant here to match the example in the preceding paragraph.   

Hmmm, and maybe add a note that mentions that the `logadm`-like use-case.  Or, add it as a second example.

Reply:
Here is the revision:
<pre>
logging/bin/user.sh CREATE --user <i>user_name</i> --password <i>password</i> 
--namespace <i>namespace</i> --tenant <i>tenant</i>
</pre>

Where: 

- *user_name* is "chen".
- *password* is the password to sign in to Kibana as the user "chen".
- *namespace* is "mynamespace".
- *tenant* is "mytenant1".

**Note:** When you specify `_all_` for *namespace* and *tenant*, the user acquires
the same level of access as a `logadm` user.